src/app/pages/private/profile/profile.page.html

```diff
@@ -38,6 +38,10 @@
             <div class="tw-text-gray-400">{{ userData()?.username}}</div>
           </div>
         </ion-item>
+
+        <ion-item>
+          <app-select-language class="tw-py-2" slot="end" />
```

Comment:
it will be also nice add 'Languag' test at the beginning of the line, to just give user more info about what this field represents. In that context we probably will not need 'slot="end"' property

Reply:
current implementation
![screenshot_clb_all](https://github.com/CollectiHub/front-end/assets/126793106/c20455b9-c00e-4470-9eac-40811fe2640e)

if you use the label outside the selector component:
![screenshot_clb_none-open](https://github.com/CollectiHub/front-end/assets/126793106/dd58f316-715f-45fd-8d5d-7b9cb07e5748)

if you use the label outside of the selector, using relative/absolute position:
```html
<ion-item class="tw-relative">
  <ion-label class="tw-text-xs tw-absolute">Language</ion-label>
  <app-select-language class="tw-py-2 tw-w-full" />
</ion-item>
```
![screenshot_clb_absolut](https://github.com/CollectiHub/front-end/assets/126793106/06954782-368d-438b-b08a-57573dec44ca)

### I think the correct solution is to use label inside the language selection component. The input flag to display this label

```html
<!-- select language component -->
<ion-select ...>
  <div class="tw-w-full tw-flex tw-justify-end" slot="label">
    <span *ngIf="displayLanguageLabel()" class="tw-mr-auto tw-text-xs tw-font-bold">
      {{ 'select_language.label' | translate }}
    </span>
    <span class="tw-mr-2 tw-text-sm"> {{ currentLanguageData.code | uppercase }} </span>
    <span class="fi fi-{{ currentLanguageData.flagIconCode }}"></span>
  </div>
  ...
</ion-select>

<!-- profile page-->
<ion-item>
  <app-select-language class="tw-py-2 tw-w-full" [displayLanguageLabel]="true" />
</ion-item>
```
![screenshot_clb_absolut](https://github.com/CollectiHub/front-end/assets/126793106/31cdfea5-7f3a-439f-813e-6e916d31aeda)